### ISSUE TITLE
Generated struct is not formatted when there are no internal fields

### ISSUE THREAD

**Issue by dhrubabasu** [StephenButtolph/canoto#43]
Using version 0.2.0.

This struct:

```golang
type Get struct {
	Key []byte `canoto:"bytes,1"`

	canotoData canotoData_Get
}
```

generates:

```golang
type canotoData_Get struct {
}
```

It should be formatted as:

```golang
type canotoData_Get struct {}
```

**Comment by dhrubabasu** [comment]
This may have been fixed by https://github.com/StephenButtolph/canoto/pull/38 (not included in 0.2.0)

**Comment by dhrubabasu** [comment]
Re-installed `v0.2.0` and I don't see this behavior anymore, closing.
